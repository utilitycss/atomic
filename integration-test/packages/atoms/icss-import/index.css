:import("@my-org/simple-class/module.css") {
  single: singleDeclaration;
  multiple: multipleDeclarations;
}

:root {
  --large: 4em;
}

.localClass {
  height: 100%;
}

.localComposes {
  composes: localClass;
  electrons: (fontWeightBold);

  text-decoration: underline;
}

.simpleComposes {
  composes: single;
}

.complexComposes {
  composes: single multiple localComposes;
  electrons: (black, white_hover);
}

.extend {
  composes: multiple;

  font-size: var(--large);
}
