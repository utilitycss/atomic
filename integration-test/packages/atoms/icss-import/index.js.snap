module.exports = {
  localClass: "n4oS1I",
  localComposes: "i6EUck n4oS1I dgII7d",
  simpleComposes: "",
  complexComposes: "i6EUck n4oS1I dgII7d _1RrCqF pQ96hs",
  extend: "_8F86eH"
};